addons/web/static/src/views/form/form_controller.js

```diff
@@ -319,7 +319,8 @@ export class FormController extends Component {
                 description: this.env._t("Archive"),
                 callback: () => {
                     const dialogProps = {
-                        body: this.env._t("Are you sure that you want to archive all this record?"),
+                        body: this.env._t("Are you sure that you want to archive this record?"),
```

Comment:
@amdi-odoo Hello, So, from one of your last PR, you only have to update translations file if targeting stable, was that it? :) You can green if so ;)